website/src/components/doc-page/doc-page-navigation.tsx

```diff
@@ -333,7 +345,7 @@ export const Navigation = styled.nav<{ height: string; show: boolean }>`
   `)}
 `;
 
-const ProductSwitcherButton = styled.button`
+const ProductSwitcherButton = styled.button<{ fullWidth?: number }>`
```

Comment:
Readonly. Why number? Isn't it rather a boolean?

Reply:
Woops, yeah of course. :D